### ISSUE TITLE
handling prerecorded rosbag file

### ISSUE THREAD

**Issue by johnny-wang16** [WKDSMRT/realsense-gstreamer#17]
Hello, I was wondering if the plugin supports streaming from prerecorded rosbag files. Thank you very much.

**Comment by TimConnelly** [comment]
No, we don't support streaming rosbag files. That would fit into a different source plugin.
